packages/web3-eth-accounts/types/index.d.ts

```diff
@@ -0,0 +1,81 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>, Samuel Furter <samuel@ethereum.org>
+ * @date 2018
+ */
+
+import {
+    AccountsBase,
+    SignedTransaction,
+    TransactionConfig,
+    WalletBase
+} from 'web3-core';
+
+export class Accounts extends AccountsBase {}
```

Comment:
Why are the Accounts and Wallet types in the web3-core package?

Reply:
so they can be shared across `web3-eth-accounts` `web3-eth-ens` without repeating them as they both shared in web3-core packages. We should not ever repeat stuff if possible.